src/unibasicTemplates/subroutines/external/find.njk

```diff
@@ -31,13 +34,20 @@ if udoGetProperty(options, 'skip', skip, type) then
   skip = 0
 end
 
+* change date.format to 2
+execute 'date.format 2'
+
 * execute the query
 udtexecute queryCommand returning errorCodes
 if @system.return.code lt 0 then
   call error_handler(ERROR_QUERY, output)
+  execute 'date.format ':systemDateFormat
   go returnFromSub
 end
 
+* set date.format back
+execute 'date.format ':systemDateFormat
```

Comment:
You could like set this back immediately after executing the query (and before the `@system.return.code` check). That way you wouldn't need to set it both when there was an error detected and without. That would also leave the least amount of lines of code between the initial setting and the reverting back.